scripts/end_to_end_test/run_end_to_end_test.py

```diff
@@ -413,6 +433,133 @@ def add_node(parameters, test_instance):
     test_instance.start_node(index)
 
 
+def create_wealth(parameters, test_instance):
+    nodes = parameters["nodes"]
+    amount = parameters["amount"]
+
+    for node_index in nodes:
+        node_host = "localhost"
+        node_port = test_instance._nodes[node_index]._port_start
+
+        api = LedgerApi(node_host, node_port)
+
+        # create the entity from the node's private key
+        entity = Entity(get_nodes_private_key(test_instance, node_index))
+        api.sync(api.tokens.wealth(entity, amount))
+
+
+def create_contract(parameters, test_instance):
+    from smart_contract_tests.synergetic_utils import ContractTestHelper
```

Comment:
Perhaps put this import at the top of the tests (?)

Reply:
This might be confusing naming due to this only being valid for creating synergetic contracts.